homeassistant/components/calendar/__init__.py

```diff
@@ -156,23 +162,26 @@ def _get_date(date):
 
         summary = self.data.event.get('summary', '')
 
-        # check if we have an offset tag in the message
+        # check if we have an offset tag in the message of the event
         # time is HH:MM or MM
         reg = '{}([+-]?[0-9]{{0,2}}(:[0-9]{{0,2}})?)'.format(self._offset)
```

Comment:
While you're at it - this should be compiled into a regex once in the constructor.